Utils/Assertions/verify.js

```diff
@@ -25,4 +30,12 @@ export default class Verify {
             }
         }
     }
+
+    async theToastMessage() {
+        return {
+            showsToastMessage: async (text) => {
+                await expect(this.page.locator(pageSels.homePage.toastMessage)).toHaveText(text)
+            }
```

Comment:
make it modularised rather than hard coded.

Reply:
Okay